core/src/main/java/org/jenkins/ui/icon/BuildStatusIcon.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2021 CloudBees, Inc.
```

Comment:
🤔

Reply:
Felix created it, I just added the license header